ramls/traits/queriable.raml

```diff
@@ -26,8 +26,6 @@ queryParameters:
   sort:
     displayName: Sort options
     type: string
-    enum: [name, relevance]
-    description: Option by which results are sorted
+    description: Option by which results are sorted. Possible values are relevance, name.
```

Comment:
I remember there was a problem in using the enum generated code in Java although I do not know what exactly that is, but will specifying strings in enum alleviate that problem instead of removing it?